src/App.tsx

```diff
@@ -29,6 +30,22 @@ function App() {
 					<Route path="/register" element={<RegisterPage />} />
 					<Route path="/login" element={<LoginPage />} />
 					<Route path="/cart" element={<CartPage />} />
+					<Route
+						path="/electronics"
```

Comment:
I suggest moving all the route path strings to a new file like `src/constants/routes.ts` and storing it as an enum.
```suggestion
			export enum Routes {
			    Register = "/register",
			    Login = "/login",
			    Cart = "/cart"
			    // ... etc
			}
			
			// and access this way
			<Route path={Paths.Login} element={<LoginPage />} />
```